build.gradle

```
@@ -293,6 +293,9 @@ android {
       dimension = 'brand'
       applicationId = 'org.medicmobile.webapp.mobile.surveillance_covid19_kenya'
     }
+    covid_moh_mali {
+      applicationId = "org.medicmobile.webapp.mobile.covid_moh_mali"
```

We now need to add the dimension too:
```suggestion
    covid_moh_mali {
      dimension = 'brand'
      applicationId = "org.medicmobile.webapp.mobile.covid_moh_mali"
```

---

Thank you @binokaryg !